.github/workflows/actionlint.yml

```diff
@@ -6,6 +6,13 @@ on:
     branches:
       - main
 
+# Cancels all previous workflow runs for pull requests that have not completed.
+concurrency:
+    # The concurrency group contains the workflow name and the branch name for pull requests
+    # or the commit hash for any other events.
+    group: ${{ github.workflow }}-${{ github.event_name == 'pull_request' && github.head_ref || github.sha }}
+    cancel-in-progress: true
+    
```

Comment:
The same thing applies here as in the frontend repository: https://github.com/WordPress/openverse-frontend/pull/1881#pullrequestreview-1130808296

We can remove this bit and then this PR will be good to go :slightly_smiling_face: 

```suggestion
```

Reply:
Done